qdrant_docs_mcp/tools/importer.py

```diff
@@ -0,0 +1,379 @@
+import argparse
+import contextlib
+import importlib.resources
+import os
+import subprocess
+import tempfile
+from collections.abc import Generator
+from dataclasses import dataclass
+from itertools import chain
+from pathlib import Path
+
+import requests
+import rich
+import semver
+from mcp_server_qdrant.embeddings.fastembed import FastEmbedProvider
+from qdrant_client import QdrantClient, models
+from rich.progress import track
+
+from qdrant_docs_mcp.tools.extractor import extract
+from qdrant_docs_mcp.tools.models import (
+    Library,
+    LibraryConfig,
+    PartialSnippet,
+    Snippet,
+    SourceConfig,
+    SourceType,
+    _VersionType,
+    get_default_config,
+)
+
+
+def _get_library_by_name(name: str) -> Library:
+    data = importlib.resources.read_text("qdrant_docs_mcp", f"libraries/{name}.json")
```

Comment:
I am getting a deprecation warning on `read_text` (I am using python 3.11)
Can we just open the file with regular `with open(...) as f:`?